Makefile

```diff
@@ -0,0 +1,17 @@
+SHELL=/bin/sh -e
+// Add list of charts: "chart1" "chart2" ...
+CHARTS := "spring-boot-example-app"
+DESTINATION_FOLDER := "charts/"
+
+.PHONY: release
+release:
+	@for chart in $(CHARTS) ; do \
+		chartFolder=$(PWD)/repository/$$chart; \
+		chartVersion=`grep '^version:' $$chartFolder/Chart.yaml | awk '{print $2}'`; \
+		echo Chart $$chart - $$chartVersion ; \
+		helm package $$chartFolder -d $(DESTINATION_FOLDER) ; \
+	done
+
+	cd $(DESTINATION_FOLDER)
+	helm repo index --url http://snowdrop.github.io/helm --merge index.yaml .
```

Comment:
Is the plan to publish the index.yml file under snowdrop/helm github repo ?

Reply:
Yes. This is also how the redhat helm repository works. 
We don't need to do anything special to publish it. Using Github Page (https://github.com/snowdrop/helm/settings/pages) automatically publish the file `index.yml` at the root project